docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,122 @@
+Some examples of test failure causes include:
+
+- Happy path: There is now a row in the table → you have detected the thing you were trying to detect!
+- The table no longer exists since one or more input data sources were deleted or inaccessible
+- Dataset schema changes, such as in the column or datatype
+- The seed in the test is different from what’s in your warehouse
+- Code releases that caused a breaking change
+
+Note: Make sure you have write permission to create a new table in your data warehouse, otherwise you may run into errors while trying to run your own tests.
+
+By default, project tests are configured to fail if the table has one or more rows. You can also modify the passing condition of your test through advanced options.
+
+## Configure a test
+
+You can configure your own passing condition of your project test.
+
+You can use the configurations to help decide which tables to focus on if you have too many with failed rows. Or conversely, you might care less about cases with multiple failed rows. For example, you might have a customer churn use case where you have 50 complaints from one customer. Configuring your test can help you identify and take action, such as assigning more resources to assist that customer.
+
+To configure a test:
+
+1. Click on the test to open the test details.
+
+2. Enter conditional values for the following options:
+
+   ![Configure a test](img/project-test-config.png)
+
+   - **A** **Failure Calculation** - Sets the failure condition used to run against the test result. You can use any function that operates on a column or multiple columns.
+   - **B** **Limit**- Sets the maximum count of all failed rows queried by the test condition. You can set the limit to save resources and time by having the test stop its query as soon as it encounters a certain number of failed rows.
+   - **C** **Severity** - Determines whether the failure of the test returns an error or warning. The severity operates from the highest priority selection, error, to the lowest, warning. So if you select error, then the test first checks for errors. If it doesn’t find any, then it then checks for warnings. If you select warning, then the test only checks for warnings. If you don’t select a severity, then error is chosen by default.
+   - **D** **Error If** and **E** **Warning If** - Sets the number of failed rows to determine a failed test. Depending on the selected severity, your test only returns a failed test for error checks. Warning won’t return a failed test.
+
+3. Click **Save**.
+
+## Schedule a test
+
+When scheduling your project, you can opt in to run a test along with the project or Model. Scheduling a test allows you to ensure on a daily basis that your data is correct. You can only schedule a test with a project or Model. You cannot schedule one individually.
+
+To Schedule your project to run with tests:
+
+1. Under the Project Browser, click **Add Job**.
+
+2. Select **Run tests** in the Model Gem in your Job.
```

Comment:
I'm confused by the phrase "in the Model Gem in your Job." 
I guess I'm not sure how you got to this screenshot.

Reply:
I added additional steps to get there.

Reply:
I'm not sure what additional steps were added, but this is also shown in the video, so LGTM. 

Honestly the product is confusing here, because you have two opportunities to select the model that will be tested. I suggest filing an Asana for this.